app/src/main/res/layout/activity_booking_detail.xml

```diff
@@ -25,472 +25,480 @@
+                        </LinearLayout>
 
                         <LinearLayout
                             android:layout_width="match_parent"
                             android:layout_height="wrap_content"
-                            android:layout_marginLeft="@dimen/dp_12"
-                            android:layout_marginStart="@dimen/dp_12"
-                            android:background="@drawable/bg_details"
-                            android:orientation="vertical"
-                            android:paddingBottom="@dimen/dp_5"
+                            android:layout_marginBottom="@dimen/dp_24"
+                            android:orientation="horizontal"
                             >
 
-                            <TextView
+                            <ImageView
+                                android:layout_width="@dimen/dp_24"
+                                android:layout_height="@dimen/dp_24"
+                                android:layout_gravity="center"
+                                android:contentDescription="@null"
+                                android:scaleType="fitCenter"
+                                android:tint="@android:color/holo_red_light"
+                                bind:srcCompat="@drawable/ic_calendar"
+                                />
+
+                            <LinearLayout
                                 android:layout_width="match_parent"
                                 android:layout_height="wrap_content"
-                                android:layout_marginBottom="@dimen/dp_5"
-                                android:text="@string/title_appointment"
-                                android:textColor="@color/colorAccent"
-                                android:textSize="@dimen/sp_15"
+                                android:layout_marginLeft="@dimen/dp_12"
+                                android:layout_marginStart="@dimen/dp_12"
+                                android:background="@drawable/bg_details"
+                                android:orientation="vertical"
+                                android:paddingBottom="@dimen/dp_5"
+                                >
+
+                                <TextView
+                                    android:layout_width="match_parent"
+                                    android:layout_height="wrap_content"
+                                    android:layout_marginBottom="@dimen/dp_5"
+                                    android:text="@string/title_appointment"
+                                    android:textColor="@color/colorAccent"
+                                    android:textSize="@dimen/sp_15"
+                                    />
+
+                                <RelativeLayout
+                                    android:layout_width="match_parent"
+                                    android:layout_height="wrap_content"
+                                    >
+
+                                    <TextView
```

Comment:
WARNING: [TooDeepLayout] `activity_booking_detail.xml` has more than 10 levels, bad for performance
 Layouts with too much nesting is bad for performance. Consider using a flatter layout (such as `RelativeLayout` or `GridLayout`).The default maximum depth is 10 but can be configured with the environment variable `ANDROID_LINT_MAX_DEPTH`.